### DISCUSSION TITLE
I See the potential, just don’t know how to use it

### DISCUSSION THREAD

**Discussion by Droyde01** [leap71/PicoGK#55] [General]
My name is Diego,I build boats parts with vcarve and i see the potential in using this to build very interesting projects, I know basic coding, i created using the documentation leap71 an environment to use PicoGK in my computer, i was able to run the example project, but i don't know how to create my own project and change the code of the program to make it work, please someone help 🙏🏻, thanks for your time

**Comment by Droyde01** [comment]
im willing to learn everything i need, if you can recommend books, videos , im a fast learner, and i have a dream to accomplish, it can sound cringe but is true

**Comment by LinKayser** [comment]
Hi @Droyde01 — one of the resources I created for this is my book Coding for Engineers. I am writing it chapter-by-chapter. Maybe it helps you getting started:

https://picogk.org/coding-for-engineers/

Best,
Lin
